apps/andi/lib/andi/input_schemas/input_converter.ex

```diff
@@ -345,6 +362,42 @@ defmodule Andi.InputSchemas.InputConverter do
     |> Map.update(:url, nil, &remove_query_params_from_url/1)
   end
 
+  defp cast_keys_to_atom_in_map_recursively(map) do
```

Comment:
If the key is a string, cast to atom.
-then-
check if the value is a map or list and repeat if needed